README.md

```diff
@@ -9,3 +9,30 @@ This is a project by us, for us, based on the things we like. If you like some o
 ## License
 
 This project is licensed under a 2-clause BSD license.
+
+## Building
+
+The easiest way to build is to use `flatpak-builder` to create a flatpack dev environment.
+
+On a Debian-based system like Ubuntu, the steps will look like below:
+
+```sh
+# Install flatpak if it's not already
+sudo apt install flatpak flatpak-builder
+
+# (Optional) Make sure that your XDG_DATA_DIRS env variable points to the proper flatpak directories
+echo "export XDG_DATA_DIRS=$XDG_DATA_DIRS:/var/lib/flatpak/exports/share:$HOME/.local/share/flatpak/exports/share" >> ~/.bashrc
```

Comment:
I'm super not sure about this, the build started failing for me unless my XDG_DATA_DIRS were `export XDG_DATA_DIRS=/var/lib/flatpak/exports/share:/home/andrew/.local/share/flatpak/exports/share:/usr/share/gnome:/usr/share/ubuntu:/usr/share:/usr/local/share` ... so I'm worried that having an XDG_DATA_DIRS that doesn't have 
`/usr/share/gnome:/usr/share/ubuntu:/usr/share:/usr/local/share` in it is frought with peril for Ubuntu at least

Reply:
Using the current construct seems to work great :D

Reply:
... after messing around with this, all I needed to do was relog after installing flatpak :|